src/wl_window.c

```diff
@@ -469,13 +491,25 @@ int _glfwPlatformCreateCursor(_GLFWcursor* cursor,
 
 int _glfwPlatformCreateStandardCursor(_GLFWcursor* cursor, int shape)
 {
-    // TODO
-    fprintf(stderr, "_glfwPlatformCreateStandardCursor not implemented yet\n");
-    return GLFW_FALSE;
+    struct wl_cursor* standard_cursor;
+
+    standard_cursor = wl_cursor_theme_get_cursor(_glfw.wl.cursorTheme,
+                                                 translateCursorShape(shape));
+    if (!standard_cursor) {
+        fprintf(stderr, "Wayland: Standard cursor not found\n");
```

Comment:
Use `_glfwInputError(GLFW_PLATFORM_ERROR, "…")` instead of fprintf, to give the message to the application instead.